package-support-policy.md

```diff
@@ -0,0 +1,372 @@
+# Package support policy for maintenance-packages
+
+As it has been described in the [.NET Release policies](https://github.com/dotnet/core/blob/main/release-policies.md) document, .NET library packages provide additional functionality that complements the .NET libraries and application frameworks like ASP.NET Core. The library packages have the same [support model as the .NET versions](https://dotnet.microsoft.com/en-us/platform/support/policy) that they target.
+
+For example, [Microsoft.Extensions.Logging](https://www.nuget.org/packages/Microsoft.Extensions.Logging/#supportedframeworks-body-tab) supports multiple .NET versions via the frameworks it targets.
+
+Our packages support any target framework versions that are currently in-support. You can find out which frameworks are supported by a specific package by visiting its NuGet page and navigating to the frameworks tab.
+
+Additionally, you can find out which target frameworks are currently in-support by visiting the following pages:
+
+- .NET: https://dotnet.microsoft.com/en-us/platform/support/policy
+- .NET Framework: https://dotnet.microsoft.com/en-us/learn/dotnet/what-is-dotnet-framework
```

Comment:
Should we link to https://learn.microsoft.com/en-us/lifecycle/products/microsoft-net-framework?

Reply:
I guess that is the document that would make more sense to link to, since it's the one that contains the support policy and also has a link to the lifecycle FAQ.

I originally opted for the [what-is-dotnet-framework](https://dotnet.microsoft.com/en-us/learn/dotnet/what-is-dotnet-framework) doc because it lists the supported versions, the out of support versions, and also a link to the support policy at the end.

I'm not strongly inclined to any option, so if you strongly feel like the support policy doc is the one that needs to be linked directly, let me know and I'll apply the suggestion below.

Or we can do it democratically 😄 . @ViktorHofer what do you think? We need your tie-breaking vote.


```suggestion
- .NET Framework: https://learn.microsoft.com/en-us/lifecycle/products/microsoft-net-framework
```